projects/user_publication/publications.py

```diff
@@ -13,22 +13,22 @@
 from projects.user_publication import gscholar, scopus, semantic_scholar, utils
 from projects.user_publication.utils import PublicationUtils, update_progress
 from django.db import transaction
-from celery import shared_task
+from celery.decorators import task
 
 LOG = logging.getLogger(__name__)
 
 
-@shared_task(bind=True)
+@task(bind=True)
 def import_pubs_scopus_task(self):
     return import_pubs_task(self, "scopus")
 
 
-@shared_task(bind=True)
+@task(bind=True)
 def import_pubs_semantic_scholar_task(self):
     return import_pubs_task(self, "semantic_scholar")
 
 
-@shared_task(bind=True)
+@task(bind=True)

```

Comment:
This seems fine, but what's the difference between "task" and "shared task" here?